src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import userDefault from '../../../../../../../../assets/user.png';
+import './MakeComment.css';
+
+const MakeComment = props => 
```

Comment:
use function and destructuring to be consistent.

Reply:
Changed in #2